ginicoeff.m

```diff
@@ -42,6 +42,7 @@
 
 % Author: Oleg Komarov (oleg.komarov@hotmail.it)
 % Tested on R14SP3 (7.1) and on R2009b
+% 18 dec 2015 - Fix comment on default sample correction
```

Comment:
Minor thing, the history is in asceding order, so latest change goes to the bottom.